test/BenchmarkMeasurementContainerTest.cpp

```diff
@@ -57,8 +51,8 @@ TEST(BenchmarkMeasurementContainerTest, ResultEntry) {
 }
 
 TEST(BenchmarkMeasurementContainerTest, ResultGroup) {
-  // The function should just wait 0.01 seconds.
-  constexpr auto waitTime = 10ms;
+  // The function should just wait 0.1 seconds.
+  constexpr auto waitTime = 100ms;
```

Comment:
Same here, that was a deliberate change.

Reply:
Done, see [0fb73ef](https://github.com/ad-freiburg/qlever/pull/1134/commits/0fb73ef2f1539ceec21e1eaa5032db1c22806f40).